### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by carlosnewmusic** [bartoszek/AUR-blender-git#1]
with -m75
```
ninja: invalid option -- 'm'
usage: ninja [options] [targets...]

if targets are unspecified, builds the 'default' target (see manual).

options:
  --version      print ninja version ("1.10.2")
  -v, --verbose  show all command lines while building

  -C DIR   change to DIR before doing anything else
  -f FILE  specify input build file [default=build.ninja]

  -j N     run N jobs in parallel (0 means infinity) [default=9 on this system]
  -k N     keep going until N jobs fail (0 means infinity) [default=1]
  -l N     do not start new jobs if the load average is greater than N
  -n       dry run (don't run commands but act like they succeeded)

  -d MODE  enable debugging (use '-d list' to list modes)
  -t TOOL  run a subtool (use '-t list' to list subtools)
    terminates toplevel options; further flags are passed to the tool
  -w FLAG  adjust warnings (use '-w list' to list warnings)
==> ERROR: Se produjo un fallo en build().
    Cancelando...
```
without any modifier
`ptxas info    : Function properties for _Z32noise_musgrave_hetero_terrain_4d6float4ffff
    0 bytes stack frame, 148 bytes spill stores, 160 bytes spill loads
ptxas info    : Function properties for _Z35shadow_blocked_transparent_all_loopP13KernelGlobalsP10ShaderDataS2_P9PathStatejP3RayP12IntersectionjP6float3
    0 bytes stack frame, 646 bytes spill stores, 928 bytes spill loads
ptxas info    : Function properties for _Z38noise_musgrave_hybrid_multi_fractal_4d6float4fffff
    0 bytes stack frame, 132 bytes spill stores, 152 bytes spill loads
ptxas info    : Function properties for _Z41kernel_volume_stack_update_for_subsurfaceP13KernelGlobalsP10ShaderDataP3RayP11VolumeStack
    0 bytes stack frame, 116 bytes spill stores, 160 bytes spill loads
ptxas info    : Function properties for _Z42kernel_branched_path_surface_connect_lightP13KernelGlobalsP10ShaderDataS2_P9PathState6float3fP12PathRadiancei
    0 bytes stack frame, 458 bytes spill stores, 464 bytes spill loads
ptxas info    : Function properties for _Z6svm_aoP13KernelGlobalsP10ShaderData6float3P9PathStatefii
    0 bytes stack frame, 72 bytes spill stores, 84 bytes spill loads
ptxas info    : Function properties for _Z9svm_bevelP13KernelGlobalsP10ShaderDataP9PathStatefi
    0 bytes stack frame, 332 bytes spill stores, 388 bytes spill loads
ninja: build stopped: subcommand failed.
==> ERROR: Se produjo un fallo en build().
    Cancelando...
`

**Comment by bartoszek** [comment]
Post full build log, or at least lines starting with `FAIL`

`-m` is a specific switch available in `ninja-mem` package.

**Comment by carlosnewmusic** [comment]
> `FAIL`

```
FAILED: intern/cycles/kernel/shaders/node_scatter_volume.oso 
cd /var/tmp/pamac-build-user/blender-git/src/build/intern/cycles/kernel/shaders && /opt/osl/bin/oslc -q -O2 -I"/var/tmp/pamac-build-user/blender-git/src/blender/intern/cycles/kernel/shaders" -I"/opt/osl/share/OSL/shaders" -o /var/tmp/pamac-build-user/blender-git/src/build/intern/cycles/kernel/shaders/node_scatter_volume.oso /var/tmp/pamac-build-user/blender-git/src/blender/intern/cycles/kernel/shaders/node_scatter_volume.osl
/opt/osl/bin/oslc: error while loading shared libraries: libclang-cpp.so.11: cannot open shared object file: No such file or directory
```
How can I apply the m modifier correctly so as not to fill all the memory in the compilation, or put it in the package so as not to block systems?

**Comment by bartoszek** [comment]
Looks like partially updated system 😉 `libclang-cpp` is provided in a dependency of `osl

**Comment by carlosnewmusic** [comment]
> Looks like partially updated system wink `libclang-cpp` is provided in the `clang` package a dependency of `osl`
even updating the system and reinstalling that package, it still fails to compile.
https://pastebin.com/rixw2Dnr

**Comment by bartoszek** [comment]
Unable to view your pastebin... 😕 
![image](https://user-images.githubusercontent.com/1213348/116377325-5b069300-a811-11eb-8130-a8c158ed1476.png)
Please just drop it onto the comment box and it will be automatically attached, just need to name it something_something.txt

**Comment by carlosnewmusic** [comment]
> > Looks like partially updated system wink `libclang-cpp` is provided in the `clang` package a dependency of `osl`
> > even updating the system and reinstalling that package, it still fails to compile.
> > https://pastebin.com/rixw2Dnr
fixed the link :)

**Comment by bartoszek** [comment]
Did the issue persist, or were you able to resolve it?

**Comment by carlosnewmusic** [comment]
> Did the issue persist, or were you able to resolve it?
no, I haven't solved it, now I get another error, although in this one
![imagen](https://user-images.githubusercontent.com/19379457/120375404-104dce80-c31b-11eb-914a-554c5f2171a9.png)
